src/pyproject_fmt/formatter/__init__.py

```diff
@@ -29,6 +29,9 @@ def format_pyproject(conf: Config) -> str:
     :return: the formatted text
     """
     parsed: TOMLDocument = parse(conf.toml)
+    overrides = parsed.get("tool", {}).get("pyproject_fmt", {})
```

Comment:
I think we should use - instead of _ in the name.

Reply:
4cb27a7ada55c60beab6c35c70134139ea80ec8c